chips/rp2040/src/chip.rs

```diff
@@ -176,6 +181,13 @@ impl<'a> Rp2040DefaultPeripherals<'a> {
 impl InterruptService for Rp2040DefaultPeripherals<'_> {
     unsafe fn service_interrupt(&self, interrupt: u32) -> bool {
         match interrupt {
+            interrupts::PIO0_IRQ_0 => {
+                // As the PIO HIL doesn't provide any support for interrupts, they are
```

Comment:
What does PIO HIL refer to here?

Reply:
Completely missed this, thank you. Put this as a placeholder probably at some point and forgot to change it - it is just the PWM comment changed to PIO. IRQs in PIO work as registers - not sure how an interrupt handler would work though, as the user/creator of the program would use them as they wish.

Reply:
Either way I think we should remove the reference to a nonexistent HIL. I think it is ok if figuring out how interrupts work for PIO is left for the first PR that actually uses them